bibliopixel/drivers/network.py

```diff
@@ -56,8 +56,7 @@ def _send_packet(self):
             log.error(error)
             raise IOError(error)
 
-    def set_brightness(self, brightness):
-        super().set_brightness(brightness)
+    def set_hardware_brightness(self, brightness):
```

Comment:
Not a fan of this name... there may be no hardware at the other end of the network.